io/src/ensenso_grabber.cpp

```diff
@@ -373,8 +375,10 @@ pcl::EnsensoGrabber::grabSinglePairOfImages (boost::shared_ptr<PairOfImages> &im
     images->second.data.resize (width * height * sizeof (float));
     images->first.encoding = images->second.encoding = getOpenCVType (channels, bpe, isFlt);
     images->first.is_bigendian = false;
-    images->first.step = 0;
-
+    images->first.step = images->first.width * sizeof (uint8_t) * 3;
+    images->second.encoding = images->second.encoding = getOpenCVType (channels, bpe, isFlt);
+    images->second.is_bigendian = false;
+    images->second.step = images->first.width * sizeof (uint8_t) * 3;
```

Comment:
Are you sure **every** image is producing element pixels of 24bytes? Because I'm seeing that there's a variable called `channels` in the code.